### PR TITLE
Update janitor to delete orphan LBs

### PR DESCRIPTION
**Pull request by nareddyt**
Find and delete all orphan target pools in `us-central1`

### CODE REVIEW

File: prow/janitor.sh [hunk 1]
```diff
@@ -112,3 +108,33 @@ for service in $ENDPOINTS_SERVICES ; do
 
 done
 echo "Done cleaning up Endpoints Services"
+
+# Load balancer target pools
+# Source: https://gist.github.com/prasvats/b2a4e33ad12b40191dd9d7e222d1abde/
```
Comment (shuoyang2016) [inline prow/janitor.sh:113]: Just want to make sure the license etc is fine?

**Review by shuoyang2016 — APPROVED** [review]
Great. Thanks, Teju!

Anyway to cleanup backend service, forwarding rules etc?

File: prow/janitor.sh (hunk 1 above)
Comment (nareddyt) [reply prow/janitor.sh:113]: Unfortunately it is not licensed, maybe we cannot use it then.

**Comment by nareddyt** [comment]
> Great. Thanks, Teju!
> 
> Anyway to cleanup backend service, forwarding rules etc?

I could not find an easy way to do this, requires writing a more complex script. We'll see how often we run into the quota issue

**Pull request closed without merging**